build/media-suite_compile.sh

```diff
@@ -582,6 +582,23 @@ if [[ $ffmpeg != "no" ]] && enabled libsoxr; then
     fi
 fi
 
+if [[ $ffmpeg != "no" ]] && enabled libcodec2 && do_pkgConfig "codec2 = 0.7"; then 
+    _check=(libcodec2.a codec2.pc) 
+	if do_wget -h 0695bb93cd985dd39f02f0db35ebc28a98b9b88747318f90774aba5f374eadb2 \
+            "https://freedv.com/wp-content/uploads/sites/8/2017/10/codec2-0.7.tar.xz"; then
+        do_uninstall include/codec2 "${_check[@]}"
+		touch cmake/GetPrerequisites.cmake
+        sed -i 's|include(${CMAKE_SOURCE_DIR}/cmake/GetPrerequisites.cmake)||' cmake/GetDependencies.cmake.in
+        sed -i 's|get_prerequisites(${CODEC2_DLL} _deps 1 0 "" "")||' cmake/GetDependencies.cmake.in
+		do_cmake -G "MSYS Makefiles"
+        do_makeinstall
```

Comment:
Unless this doesn't work with ninja, replace this with
```
log make ninja
log install ninja install
```

Reply:
Doesn't work for me at all: 
ninja: error: loading 'build.ninja': Das System kann die angegebene Datei nicht finden.
Which means 'build.ninja' cannot be found. 

I works as per my patch in 32bit and 64bit, tested.

Reply:
You'd need to remove the `-G "MSYS Makefiles"` first so the ninja files are generated.
If ninja works, it's way better than using make.